qmt/physics_constants.py

```diff
@@ -94,12 +79,17 @@ def canonicalize(expr, base=None):
 
 
 def cancel(expr):
```

Comment:
I'm not sure. This is only tested for sympy 1.0 (which is the version we're using in our conda environment). If we upgrade actually some things in QMS breaks as well. We'd have to do it at some point, but I was planning to upgrade all the sympy stuff as part of a broader packages upgrade to python 3.7

Reply:
I see. The reason I ask is because the whole to_float thing was designed to circumvent the change of api in sympy units. I think it should work, but we'll find it out after testing. 

See https://github.com/sympy/sympy/wiki/Release-Notes-for-1.2